{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "moodle",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "4.0 to 4.0.3, 3.11 to 3.11.9, 3.9 to 3.9.16 and earlier unsupported versions"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "A remote code execution risk when restoring backup files originating from Moodle 1.9 was identified."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "Remote code execution",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2022-09-30T16:29:22.000Z",
                "orgId": "92fb86c3-55a5-4fb5-9c3f-4757b9e96dc5",
                "shortName": "fedora"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2128147"
                },
                {
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://moodle.org/mod/forum/discuss.php?d=438393"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "patrick@puiterwijk.org",
                    "ID": "CVE-2022-40314",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "moodle",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "4.0 to 4.0.3, 3.11 to 3.11.9, 3.9 to 3.9.16 and earlier unsupported versions"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "A remote code execution risk when restoring backup files originating from Moodle 1.9 was identified."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "Remote code execution"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://bugzilla.redhat.com/show_bug.cgi?id=2128147",
                            "refsource": "MISC",
                            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2128147"
                        },
                        {
                            "name": "https://moodle.org/mod/forum/discuss.php?d=438393",
                            "refsource": "MISC",
                            "url": "https://moodle.org/mod/forum/discuss.php?d=438393"
                        }
                    ]
                }
            }
        },
        "adp": [
            {
                "providerMetadata": {
                    "orgId": "af854a3a-2127-422b-91ae-364da2661108",
                    "shortName": "CVE",
                    "dateUpdated": "2024-08-03T12:14:40.012Z"
                },
                "title": "CVE Program Container",
                "references": [
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_transferred"
                        ],
                        "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2128147"
                    },
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_transferred"
                        ],
                        "url": "https://moodle.org/mod/forum/discuss.php?d=438393"
                    }
                ]
            },
            {
                "problemTypes": [
                    {
                        "descriptions": [
                            {
                                "type": "CWE",
                                "cweId": "CWE-502",
                                "lang": "en",
                                "description": "CWE-502 Deserialization of Untrusted Data"
                            }
                        ]
                    }
                ],
                "metrics": [
                    {
                        "cvssV3_1": {
                            "scope": "UNCHANGED",
                            "version": "3.1",
                            "baseScore": 9.8,
                            "attackVector": "NETWORK",
                            "baseSeverity": "CRITICAL",
                            "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
                            "integrityImpact": "HIGH",
                            "userInteraction": "NONE",
                            "attackComplexity": "LOW",
                            "availabilityImpact": "HIGH",
                            "privilegesRequired": "NONE",
                            "confidentialityImpact": "HIGH"
                        }
                    },
                    {
                        "other": {
                            "type": "ssvc",
                            "content": {
                                "timestamp": "2025-05-20T18:23:16.446219Z",
                                "id": "CVE-2022-40314",
                                "options": [
                                    {
                                        "Exploitation": "none"
                                    },
                                    {
                                        "Automatable": "yes"
                                    },
                                    {
                                        "Technical Impact": "total"
                                    }
                                ],
                                "role": "CISA Coordinator",
                                "version": "2.0.3"
                            }
                        }
                    }
                ],
                "title": "CISA ADP Vulnrichment",
                "providerMetadata": {
                    "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
                    "shortName": "CISA-ADP",
                    "dateUpdated": "2025-05-20T18:23:40.483Z"
                }
            }
        ]
    },
    "cveMetadata": {
        "assignerOrgId": "92fb86c3-55a5-4fb5-9c3f-4757b9e96dc5",
        "assignerShortName": "fedora",
        "cveId": "CVE-2022-40314",
        "datePublished": "2022-09-30T16:29:22.000Z",
        "dateReserved": "2022-09-09T00:00:00.000Z",
        "dateUpdated": "2025-05-20T18:23:40.483Z",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1"
}